sqlmesh/core/snapshot/definition.py

```diff
@@ -605,6 +609,9 @@ class Snapshot(PydanticModel, SnapshotInfoMixin):
     base_table_name_override: t.Optional[str] = None
     next_auto_restatement_ts: t.Optional[int] = None
     dev_table_suffix: str = "dev"
+    table_naming_convention_: t.Optional[TableNamingConvention] = Field(
```

Comment:
When reusing the physical version (eg. FORWARD_ONLY, METADATA) we should inherit the convention of the previous snapshot just like we do the physical schema: https://github.com/TobikoData/sqlmesh/blob/main/sqlmesh/core/snapshot/definition.py#L1015